pkgs/development/python-modules/markdownsuperscript/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, markdown }:
+
+buildPythonPackage rec {
+  pname = "MarkdownSuperscript";
+  version = "2.0.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1dsx21h9hkx098d5azpw81dcz23rrgzrwlymwv7jri348q26p748";
+  };
+
+  propagatedBuildInputs = [ markdown ];
+
+  doCheck = false; # Tests depend on packages which are not available yet in nixpkgs
```

Comment:
@Zimmi48 sometimes getting the tests to work takes a lot of time and sometimes they just don't work at all. I appreciate the time you put in it, but if you notice it isn't working or going forward, then we simply merge what we have and leave it at that.